components/sup/src/error.rs

```diff
@@ -505,6 +508,12 @@ impl From<toml::de::Error> for SupError {
     }
 }
 
+impl From<serde_json::Error> for SupError {
```

Comment:
Per my comment below, I'd remove this trait implementation and the JsonParser error completely